pkg/astprinter/astprinter.go

```diff
@@ -1043,3 +1043,13 @@ func (p *printVisitor) writeFieldType(ref int) {
 	p.write(literal.SPACE)
 	p.must(p.document.PrintType(p.document.FieldDefinitionType(ref), p.out))
 }
+
+func countIndentDepth(indent []byte) int {
```

Comment:
Not fully understand what you are doing here?

In case the indent doesn't have a tab character returned value will be 0